Makefile

```diff
@@ -109,6 +111,12 @@ lint-python-files:
 	$(VIRTUAL_ENV)/bin/flake8 $(ARGUMENTS) || EXIT_STATUS=$$?; \
 	exit $${EXIT_STATUS}
 
+.PHONY: lint-fix
+lint-fix:
+	EXIT_STATUS=0; \
+	npm run lint-fix --exclude node_modules ||  EXIT_STATUS=$$?; \
+	exit $${EXIT_STATUS}
```

Comment:
this is still being tested but I found it useful for the eslint updates, for other times it doesn't make sense, it's also still being tested as it takes ages